doc/Tex/RAM_SCB_install.tex

```diff
@@ -1,180 +1,72 @@
-this happens, check for the creation of a {\tt lib} directory and several 
-\verb*l*.al files contained within.  If they exist, disregard the error 
-message and continue to the next step.
-
-Install Pspline as follows (changing the value of {\tt PREFIX} to select 
-installation directory):
-\begin{verbatim}
-make install PREFIX=~/psline_portland
-\end{verbatim}
-On machines where you will use different compilers, it is helpful to add the 
-compiler name to the name of the installation directory.
+% QUESTION: All of the information in this section is out of date. Even the NETCDF installation instructions are incorrect given that they have split it into a C version and a Fortran version. This is still easy enough to describe and walk through, but the installation of GSL from source isn't as easy. Do we really want to have this section?
 
 \subsection{Libraries From Modules}
-Well maintained clusters often use the {\tt module} interface for loading
-and unloading software packages.  Be sure to explore the available packages
-on a new system before going through the arduous task of installing from source!
+Well maintained clusters often use the {\tt module} interface for loading and unloading software packages.  Be sure to explore the available packages on a new system before going through the arduous task of installing from source!
 
 \begin{verbatim}
 module avail
 \end{verbatim}
 \noindent
-will list all available software libraries on the system.  You can load a 
-library, unload a library, and list loaded software using the module commands:
+will list all available software libraries on the system.  You can load a library, unload a library, and list loaded software using the module commands:
 
 \begin{verbatim}
 module load package_name
 module unload package_name
 module list
 \end{verbatim}
 
-As an example, here is what you would type on NASA's Pleiades cluster to get
-started with RAM-SCB:
+As an example, here is what you would type on NASA's Pleiades cluster to get started with RAM-SCB:
+% QUESTION: ^Is this even still true? Seems silly to have to keep up with changes on Pleiades in our documentation
 
 \begin{verbatim}
 module load comp/intel/10.1.021_64
 module load mpi/mpt.1.25
 module load netcdf/4.0-i10.1
-module load ncarg/4.4.2/intel
+module load gsl/2.4
 module load python/2.6.1
 \end{verbatim}
 
-Intel Fortran, MPI, NetCDF and NCAR Graphics are now loaded and ready to use.
-Place such commands your shell configuration 
-script ({\tt \verb*l~l/.cshrc} or {\tt \verb*l~l/.bashrc} depending on your 
-shell) to load modules upon login.  Do not forget to point RAM-SCB to these
-libraries as listed below.
+Intel Fortran, MPI, NetCDF and GSL are now loaded and ready to use. Place such commands your shell configuration script ({\tt \verb*l~l/.cshrc} or {\tt \verb*l~l/.bashrc} depending on your shell) to load modules upon login. Do not forget to point RAM-SCB to these libraries as listed below.
```

Comment:
This approach is okay, and I'm not going to recommend folks *don't* do it (especially if you're running on a local server), but most job schedulers on HPC launch the jobs in a new environment, so modules loaded from the `rc` file may not propagate to your job.
Note here that the module may need to be loaded in a job script on systems managed by a job scheduler? (e.g., slurm, moab, pbs pro)

Reply:
Thats a good point, I'll add the clarification